src/web/app/components/question-submission-form/question-submission-form.component.ts

```diff
@@ -94,10 +94,22 @@ export class QuestionSubmissionFormComponent implements OnInit {
               private feedbackResponseService: FeedbackResponsesService) {
     this.visibilityStateMachine =
         this.feedbackQuestionsService.getNewVisibilityStateMachine(
-            this.model.giverType, this.model.recipientType);
+          this.model.giverType, this.model.recipientType);
   }
 
   ngOnInit(): void {
+    // Sorts the receipients alphabetically
+    this.model.recipientSubmissionForms.sort((firstRecepient: FeedbackResponseRecipientSubmissionFormModel, secondRecepient: FeedbackResponseRecipientSubmissionFormModel) => {
```

Comment:
Should this sort go into a method? Was thinking of putting it in a private class method or left as a public class method?

Reply:
Since we might add more code to `ngOnInit()` function in the future, I agree that moving this into private method would be better, or else it will get a little messy.
You can do this by moving the entire block into a private method in this component and call the method in `ngOnInit()`